third_party/move/move-prover/move-abigen/src/abigen.rs

```diff
@@ -122,9 +122,7 @@ impl<'env> Abigen<'env> {
     /// Compute the ABIs of all script functions in a module.
     fn compute_abi(&self, module_env: &ModuleEnv<'env>) -> anyhow::Result<Vec<ScriptABI>> {
         // Get all the script functions in this module
-        let script_iter: Vec<_> = if module_env.is_script_module() {
```

Comment:
This needs to stay to support proper scripts, AFAICT.

Reply:
I changed the check below to work for a script module.